airflow_tecton/operators/tecton_feature_table_job_operator.py

```diff
@@ -0,0 +1,100 @@
+    """
+    An Airflow operator that launches a Tecton Feature Table ingestion job,
+    and waits for its completion.
+
+    Use this if you want to submit a Tecton job via Airflow and control
+    retries via Airflow. Each attempt of this operator creates 1 job.
+    """
+
+    template_fields: Sequence[str] = ("templates_dict", "op_args", "op_kwargs")
+    template_fields_renderers = {"templates_dict": "json", "op_args": "py", "op_kwargs": "py"}
+
+    def __init__(
+        self,
+        *,
+        conn_id: str = "tecton_default",
+        workspace: str,
+        feature_view: str,
+        online: bool,
+        offline: bool,
+        df_generator: Callable[..., pandas.DataFrame] = None,
+        op_args: Union[Collection[Any], None] = None,
+        op_kwargs: Union[Mapping[str, Any], None] = None,
+        templates_dict: Union[dict, None] = None,
+        templates_exts: Union[list[str], None] = None,
+        **kwargs,
+    ):
+        """
+
+        :param conn_id: Airflow connection ID for Tecton connection
+        :param workspace: Workspace of FeatureView
+        :param feature_view: FeatureView name
+        :param online: Whether job writes to online store
+        :param offline: Whether job writes to offline store
+        :param df_generator: A reference to an object that is callable and
+               returns pandas.DataFrame
+        :param op_args: a list of positional arguments that will get unpacked when
+            calling df_generator
+        :param op_kwargs: a dictionary of keyword arguments that will get unpacked
+            in df_generator
+        :param templates_dict: a dictionary where the values are templates that
+            will get templated by the Airflow engine sometime between
+            ``__init__`` and ``execute`` takes place and are made available
+            in df generator's context after the template has been applied. (templated)
+        :param templates_exts: a list of file extensions to resolve while
+            processing templated fields, for examples ``['.sql', '.hql']``
+        :param kwargs: Airflow base kwargs passed to BaseOperator
+        """
+        super().__init__(**kwargs)
+        self.workspace = workspace
+        self.feature_view = feature_view
+        self.online = online
+        self.offline = offline
+        self.conn_id = conn_id
+        self.job_id = None
+        if df_generator and not callable(df_generator):
+            raise Exception("`df_generator` param must be callable")
+        self.df_generator = df_generator
+        self.op_args = op_args or ()
+        self.op_kwargs = op_kwargs or {}
+        self.templates_dict = templates_dict
+        if templates_exts:
+            self.template_ext = templates_exts
+
+    def execute(self, context: Context) -> Any:
+        hook = TectonHook.create(self.conn_id)
+
+        resp = ingest_feature_table_with_pandas_df(hook, self.workspace, self.feature_view, context,
```

Comment:
i think you need to pass the retry stuff here

Reply:
not really, I set default tecton_managed_retries to False.